Azure-Samples/AI-Gateway
# Add markdownlint and EditorConfig

## Purpose

- Add markdownlint to introduce consistent markdown behavior.
- Add EditorConfig to enforce consistent file behavior.

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
1. Use VS Code (or other editor with other installs).
1. Install [markdownlint](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint).
1. Install [EditorConfig for VS Code](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig).
1. Open Markdown files in VS Code and observe rule checking.
 -->
